ui/lib/utils.ts

```diff
@@ -79,3 +79,13 @@ export function removeKind(kind: string): string {
   }
   return kind;
 }
+
+export function imageString(images: string[]) {
```

Comment:
It is common to start the name of a function with a verb (`make`, `compose`, `generate`, I don't know, whatever fits). But it's a subjective preference.